app/controllers/users.js

```diff
@@ -1,6 +1,6 @@
 const logger = require('../logger');
-const { encryptPass } = require('../helpers/users');
-const { insertUser } = require('../services/users');
+const { encryptPass, getPagination, getPaginData } = require('../helpers/users');
```

Comment:
Pagination methods sound generic. I think it should be in an upper scope, like `/helpers`.